packages/vite/src/node/plugins/asset.ts

```diff
@@ -128,15 +128,16 @@ export function checkPublicFile(
   }
 }
 
-export function fileToUrl(
+export async function fileToUrl(
   id: string,
   config: ResolvedConfig,
   ctx: PluginContext
 ) {
```

Comment:
**suggestion (non-blocking):** You could add a return type here, so the API gets more stable / if something in the body changes in the future, the developer receives a compile error

Reply:
fileToUrl is used internally. if its return type changes unexpectedly, there will be compiler errors whether or not a return type exists